src/com/dotmarketing/portlets/contentlet/business/web/ContentletWebAPIImpl.java

```diff
@@ -183,12 +185,19 @@ public String saveContent(Map<String, Object> contentletFormData,
 		if(autocommit)
 		    HibernateUtil.commitTransaction();
 
+		ContentletSystemEventUtil.getInstance().pushSaveEvent(user, cont, isNew);

```

Comment:
Are we creating events when saving any type of Contentlet?, I thought we should do it just for hosts.